README.md

```diff
@@ -938,13 +938,19 @@ To see a list of the rules, refer to the [stylelint-config-standard](https://git
 - [stylelint-webpack-plugin](https://github.com/webpack-contrib/stylelint-webpack-plugin)
 - [stylelint PostCSS plugin](https://stylelint.io/user-guide/postcss-plugin)
 - [gulp-stylelint](https://github.com/olegskl/gulp-stylelint)
+- [Stylelint plugins for your code editor](https://stylelint.io/user-guide/complementary-tools#editor-plugins)
 
 ##### Auto fix errors
 Stylelint CLI can autofix as many errors as possible. To use the autofixer append `--fix` to a stylelint command.
 ```
 stylelint "foo/*.scss" --fix
 ```
 
+You may have to install stylelint globally:
+```
+npm install stylelint -g
+```
+
```

Comment:
@srayeskyo3 yes I think this is a good note as this will confuse people

Instead of installing globally you can also add a script in `package.json` to run it as well, like:
```
"scripts": {
    "styelint": "stylelint foo/**/*.scss'",
  ...
}
```
then run with `npm run stylelint` or `yarn stylelint`

would probably be good to add a note for that as well.